libs/scheduler/aws.go

```
@@ -311,32 +330,43 @@ func (fetcher *GithubAwsTokenFetcher) GetIdentityToken() ([]byte, error) {
 
 	resp, err := httpClient.Do(req)
 	if err != nil {
+		slog.Error("Failed to fetch GitHub identity token", "error", err)
 		return nil, err
 	}
 	defer resp.Body.Close()
 	parsed := &TokenResponse{}
 	json.NewDecoder(resp.Body).Decode(parsed)
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Decoder).Decode` is not checked

(errcheck)